### PR TITLE
Complete remaining request handlers

### PR DESCRIPTION
**Pull request by LukeLarson2**
- Added routes/router.js to handle all requests
- Updated Schema to improve indexing on tables
- Updated etl.js to set id serial to highest value +1 after data upload per table
- Added request handler for post a review
- Added request handler for put on helpfulness
- Added request handler for put on report

Initial query times are still sub 10ms, post request is at 18ms latency, will work to optimize if necessary once testing in K6/deployed

Please review @pjol @justcord10

### CODE REVIEW

**Review by pjol — APPROVED** [review]

File: schema.psql [hunk 1]
```diff
@@ -21,24 +21,28 @@ CREATE TABLE reviews (
...
   value INT
 );
 
-CREATE INDEX idx_characteristics_product_id ON characteristics(product_id);
\ No newline at end of file
+CREATE INDEX idx_characteristics_product_id ON characteristics(product_id);
+CREATE INDEX idx_char_reviews_characteristic_id ON char_reviews(characteristic_id);
+CREATE INDEX idx_photos_review_id ON photos(review_id);
+CREATE INDEX idx_char_reviews_review_id ON char_reviews(review_id);
+CREATE INDEX idx_reviews_product_id_review_date ON reviews(product_id, review_date, id);
```
Comment (justcord10) [inline schema.psql:48]: added the rest of your indexes for faster responses

File: routes/router.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    `;
+    await pool.query(queryText, [id]);
+    res.status(200).send('Reported updated successfully');
+  } catch (error) {
+    console.error(error);
+    res.status(500).send('Server Error');
+  }
+});
+
+module.exports = router;
```
Comment (justcord10) [inline routes/router.js:164]: moved your routes to a router function and added in the rest of your routes.

File: schema.psql [hunk 3]
```diff
@@ -21,24 +21,28 @@ CREATE TABLE reviews (
...
   url TEXT
 );
 
 -- Create Characteristics Table
 CREATE TABLE characteristics (
-  characteristic_id INT PRIMARY KEY,
-  product_id INT REFERENCES reviews(product_id),
+  characteristic_id SERIAL PRIMARY KEY,
+  product_id INT,  -- Modify this line based on your data model
   name TEXT
```
Comment (justcord10) [inline schema.psql:33]: removed the product_id as a foreign key?

File: server.js [hunk 4]
```diff
@@ -1,44 +1,16 @@
...
 
-const pool = new Pool({
-  user: process.env.DB_USER,
-  database: process.env.DB_NAME,
-  port: process.env.DB_PORT,
-  ssl: false
-});
-
 app.use(express.json());
+app.use('/', router);
```
Comment (justcord10) [inline server.js:11]: all incoming requests should go to the router

**Review by justcord10 — APPROVED** [review]
Looks good, refactoring routes into a router file makes the server code way cleaner.

**Pull request merged**